### ISSUE TITLE
Not able to compile IOS app

### ISSUE THREAD

**Issue by Gitrock2023** [google/android-auto-companion-app#3]
Trying to create IOS app using XCode version 13.4.1 and flutter version 3.3.3 but I am getting below error:
Unable to get Xcode project information:
 2023-05-30 15:42:24.190 xcodebuild[3753:51032] Requested but did not find
 extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications
 for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in
 com.apple.dt.IDEWatchSupportCore
2023-05-30 15:42:24.190 xcodebuild[3753:51032] Requested but did not find
extension point with identifier
Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension
Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in
com.apple.dt.IDEWatchSupportCore
xcodebuild: error: Could not resolve package dependencies:
  Dependencies could not be resolved because root depends on
  'android-auto-companion-calendarsync-ios' 1.5.0..<2.0.0.
'android-auto-companion-calendarsync-ios' >= 1.5.0 cannot be used because no
versions of 'android-auto-companion-calendarsync-ios' match the requirement
1.5.1..<2.0.0 and 'android-auto-companion-calendarsync-ios' 1.5.0 contains
incompatible tools version (5.7.0).

**Comment by tapsns** [comment]
The latest release requires Swift 5.7 or later. Your Xcode is 13.4.1, but you will need Xcode 14 or later to build. If you want to use Xcode 13.4.1 then you will need to choose an earlier release that is compatible with it (check the Swift requirement in Package.swift).

**Comment by Gitrock2023** [comment]
I upgraded the XCode to 14.2 which resolved the dependency issue but I am getting another error.

    error: the following command failed with exit code 1 but produced no further output
    Validate plug-in “ProtoSourceGenerator” in package “android-auto-companion-ios”

    Result bundle written to path:
    	/var/folders/4s/wh58jt_d5gz_6r7vd9hhky5r0000gn/T/flutter_tools.ig3W2U/flutter_ios_build_temp_dirVpTRRs/temporary_xcresult_bundle


Uncategorized (Xcode): “ProtoSourceGenerator” is disabled
/Users/singhv/Library/Developer/Xcode/DerivedData/Runner-feaybrhmtllnlrdslrlderhfmocu/SourcePackages/checkouts/android-auto-companion-ios/Plugins/ProtoSourceGenerator/plugin.swift


Encountered error while archiving for device.

Seems like actual problem is with protocbuf not installing protoc and other required executable in usr/local/bin folder.
 I tried brew install protobuf but still getting below error.
/usr/local/bin/protoc: /usr/local/bin/protoc: cannot execute binary file
